core/schema/src/main/liquibase/29.0.4/changelog.xml

```diff
@@ -11,4 +11,131 @@
+        </sqlCheck>
+        <sqlCheck expectedResult="0">
+          SELECT COUNT(hypertable_name) FROM timescaledb_information.hypertables WHERE hypertable_name = 'bmp_ip_rib_log';
+        </sqlCheck>
+      </and>
+    </preConditions>
+
+    <sql>
+      SELECT create_hypertable('bmp_ip_rib_log', 'last_updated', chunk_time_interval => interval '5 day', migrate_data => true);
+    </sql>
+  </changeSet>
+
+
+  <changeSet author="aramos" id="29.0.4-bmp-stats_by-peer-create-hypertable" runAlways="true">
+    <preConditions onFail="MARK_RAN">
+      <and>
+        <sqlCheck expectedResult="1">
+          SELECT COUNT(e.extname)
+          FROM pg_catalog.pg_extension e LEFT JOIN pg_catalog.pg_description c ON c.objoid = e.oid AND c.classoid =
+          'pg_catalog.pg_extension'::pg_catalog.regclass
+          WHERE e.extname LIKE 'timescaledb';
+        </sqlCheck>
+        <sqlCheck expectedResult="0">
+          SELECT COUNT(hypertable_name) FROM timescaledb_information.hypertables WHERE hypertable_name = 'bmp_stats_by_peer';
+        </sqlCheck>
+      </and>
+    </preConditions>
+
+    <sql>
+      SELECT create_hypertable('bmp_stats_by_peer', 'interval_time', chunk_time_interval => interval '2 day', migrate_data => true);
+    </sql>
+  </changeSet>
+
+
+  <changeSet author="aramos" id="29.0.4-bmp-stats_by-asn-create-hypertable" runAlways="true">
+    <preConditions onFail="MARK_RAN">
+      <and>
+        <sqlCheck expectedResult="1">
+          SELECT COUNT(e.extname)
+          FROM pg_catalog.pg_extension e LEFT JOIN pg_catalog.pg_description c ON c.objoid = e.oid AND c.classoid =
+          'pg_catalog.pg_extension'::pg_catalog.regclass
+          WHERE e.extname LIKE 'timescaledb';
+        </sqlCheck>
+        <sqlCheck expectedResult="0">
+          SELECT COUNT(hypertable_name) FROM timescaledb_information.hypertables WHERE hypertable_name = 'bmp_stats_by_asn';
+        </sqlCheck>
+      </and>
+    </preConditions>
+
+    <sql>
+      SELECT create_hypertable('bmp_stats_by_asn', 'interval_time', chunk_time_interval => interval '2 day', migrate_data => true);
+    </sql>
+  </changeSet>
+
+
+
+  <changeSet author="aramos" id="29.0.4-bmp-stats_by-prefix-create-hypertable" runAlways="true">
+    <preConditions onFail="MARK_RAN">
+      <and>
+        <sqlCheck expectedResult="1">
+          SELECT COUNT(e.extname)
+          FROM pg_catalog.pg_extension e LEFT JOIN pg_catalog.pg_description c ON c.objoid = e.oid AND c.classoid =
+          'pg_catalog.pg_extension'::pg_catalog.regclass
+          WHERE e.extname LIKE 'timescaledb';
+        </sqlCheck>
+        <sqlCheck expectedResult="0">
+          SELECT COUNT(hypertable_name) FROM timescaledb_information.hypertables WHERE hypertable_name = 'bmp_stats_by_prefix';
+        </sqlCheck>
+      </and>
+    </preConditions>
+
+    <sql>
+      SELECT create_hypertable('bmp_stats_by_prefix', 'interval_time', chunk_time_interval => interval '2 day', migrate_data => true);
```

Comment:
Instead of having different `chunk_time_interval` for each of the tables,  We can start with default `7 day` as specified [here](https://docs.timescale.com/timescaledb/latest/how-to-guides/hypertables/best-practices/#time-intervals).

And this can be updated by user if required.